examples/pylint/C0321_multiple_statements.py

```diff
@@ -0,0 +1,6 @@
+def pos(temp):
+    """Print "positive" if <temp> is greater than 0. ELse print "negative".
```

Comment:
In general, please don't use `print` in examples. Just return something instead.